nrfe/app/nrfe_gpio.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Let's use BSD3 for now like in nrfx:
https://github.com/magp-nordic/hal_nordic/blob/master/nrfx/hal/nrf_aar.h#L1-L33